### PR TITLE
remove outline and add focus hover states to ws content selector

### PR DESCRIPTION
**Pull request by udaraweerasinghege**
Thank you @ericjim & @cchawn for the css help.

This PR removes the default outline that browsers have when an element is in :focus for ws-content-selector. Replaces it by making the elements darker when hovered/focused. This also removes the hover logic we had in our js that was previously controlling this.

image

image

image

### CODE REVIEW

File: src/form/content-selector/_ws-content-selector.scss [hunk 1]
```diff
@@ -56,6 +56,10 @@
         }
       }
 
+      &:hover:not(.selected),&:focus:not(.selected){
+        background-color: $color-gray-10;
+      }
```
Comment (SethDavenport) [inline src/form/content-selector/_ws-content-selector.scss]: did you test in IE?

Comment (udaraweerasinghege) [reply src/form/content-selector/_ws-content-selector.scss]: The website (https://wealthsimple.github.io/ws-directives/) does not work on ie o.o , the feature did work on edge though

**Comment by AlexisDeschamps** [comment]
Can probably test in IE now that the docs page is fixed.

**Comment by udaraweerasinghege** [comment]
Tested and working on ie :D

**Pull request merged**